l10n_it_reverse_charge/models/account_move.py

```diff
@@ -349,9 +350,16 @@ def button_cancel(self):
 
     def button_draft(self):
         new_self = self.with_context(rc_set_to_draft=True)
-        super(AccountMove, new_self).button_draft()
         invoice_model = new_self.env["account.move"]
         for inv in new_self:
+            # remove payments without deleting self invoice
+            inv.remove_rc_payment(delete_self_invoice=False)
+            if inv.payment_id:
```

Comment:
**Proposta**: per evitare di duplicare il codice puoi estrapolare le righe seguenti in un metodo dedicato (es. `remove_payment` da riutilizzare anche [all'interno](https://github.com/OCA/l10n-italy/blob/e702e6caf43138cfc7dd487c2f202f9e6c24745c/l10n_it_reverse_charge/models/account_move.py#L326-L330) di `remove_rc_payment`?

Reply:
Fatto, grazie

Reply:
:+1: